wiki/Contests/en.md

```diff
@@ -375,23 +369,25 @@ The **Newspaper Cup** is a mapping contest that is similar to the Pending Cup in
-| Instrumental² | ![osu!][osu!] | ![][flag_CA] [Kyumo](https://osu.ppy.sh/users/14689984) | ![][flag_US] [- ascended -](https://osu.ppy.sh/users/4564285) | ![][flag_CL] [Sunazuka Akira](https://osu.ppy.sh/users/8846632) | [#1](https://osu.ppy.sh/community/forums/topics/1274667) |
-
-¹ The Vocal category provides mappers with a choice of five different songs to be mapped: Ariabl'eyeS - Unmei no Haguruma, Yunomi - Aimai Trip, KikuoHana - Desert Theater, Kaf - Tomadoi Telepathy, and Mafumafu - SNOBBISM.
-
-² The Instrumental category provides mappers with a choice of five different songs to be mapped: Umeboshi Chazuke - Dutch Courage!, The Flashbulb - Stretch Industry, MiddleIsland - Delrio, Nagakawa Koutarou - Hakata Tonkotsu Ramens, and roop - Chatblanc.
+| Vocal[^newspaper-cup-7-vocal] | ![osu!][osu!] | ![][flag_EE] [Keqing](https://osu.ppy.sh/users/8501291) | ![][flag_US] [Cubby](https://osu.ppy.sh/users/10914582) | ![][flag_TW] [Hey lululu](https://osu.ppy.sh/users/4086497) | [#1](https://osu.ppy.sh/community/forums/topics/1274667) |
+| Instrumental[^newspaper-cup-7-instrumental] | ![osu!][osu!] | ![][flag_CA] [Kyumo](https://osu.ppy.sh/users/14689984) | ![][flag_US] [- ascended -](https://osu.ppy.sh/users/4564285) | ![][flag_CL] [Sunazuka Akira](https://osu.ppy.sh/users/8846632) | [#1](https://osu.ppy.sh/community/forums/topics/1274667) |
+
+## References
+
+[^pending-cup-8-fa]: The Featured Artist category provides mappers with a choice of two different songs to be mapped: Zekk - Re\_construct and Shuu no Hazama \[Rainshower\].
+[^pending-cup-8-vocal]: The Vocal category provides mappers with a choice of two different songs to be mapped: nonoc - memento and DUSTCELL - DERO.
+[^pending-cup-8-instrumental]: The Instrumental category provides mappers with a choice of two different songs to be mapped: onoken - Felys final remix and Nekomata Master - The Sky of Sadness.
+[^newspaper-cup-6-vocal]: The Vocal category provides mappers with a choice of two different songs to be mapped: Isekai Joucho - Kyoukankaku Obake and Kenshi Yonezu - Shunrai.
+[^newspaper-cup-6-instrumental]: The Instrumental category provides mappers with a choice of two different songs to be mapped: Morimori Atsushi - Funkotsu Saishin Casino and Fractal Dreamers - Celestial Horizon.
+[^newspaper-cup-7-vocal]: The Vocal category provides mappers with a choice of five different songs to be mapped: Ariabl'eyeS - Unmei no Haguruma, Yunomi - Aimai Trip, KikuoHana - Desert Theater, Kaf - Tomadoi Telepathy, and Mafumafu - SNOBBISM.
+[^newspaper-cup-7-instrumental]: The Instrumental category provides mappers with a choice of five different songs to be mapped: Umeboshi Chazuke - Dutch Courage!, The Flashbulb - Stretch Industry, MiddleIsland - Delrio, Nagakawa Koutarou - Hakata Tonkotsu Ramens, and roop - Chatblanc.
```

Comment:
should this page have footnotes that only pertain to a few tournaments in the listing?

Reply:
seems fine by me